web/public/js/directives/cd-action-bar/cd-action-bar-popup-item/style.less

```diff
@@ -35,27 +37,36 @@
       // TODO: Have this configurable in some way. Maybe sm, md and lg or something...
       width: 320px;
 
-      @media (max-width: @screen-xs-max) {
-        display: none;
-      }
-
-      &--mobile {
-        display: none;
-        left: 0;
-        width: 100%;
+      &--visible {
+        display: block;
 
         @media (max-width: @screen-xs-max) {
```

Comment:
this is weird, mind xplaining?
is it that you switch from __popup to __full-width-popup when the screen is xs?